apps/pdc-frontend/src/styles/openforms.scss

```diff
@@ -12,3 +12,8 @@
 .formio-editor-read-only-content {
   @include utrecht-multiline-data;
 }
+
+/* previous button alignment */
+.utrecht-form-container--openforms .openforms-toolbar .openforms-toolbar__list-item .utrecht-link--openforms {
```

Comment:
Met welk element wil je deze link uitlijnen? Is er een referentiestijl voor, bijvoorbeeld in Figma?

Het lijkt erop dat de CSS-property `justify-content: left;` niet werkt.

<img width="491" alt="Screenshot 2024-05-01 at 11 22 05" src="https://github.com/frameless/strapi/assets/53454241/4b52ed9c-11b2-4085-8766-4504a3548511">